### DISCUSSION TITLE
Account has been flagged as spam for no reason and Github support won't respond even after a month.

### DISCUSSION THREAD

**Discussion by iamPedram1** [community/community#114472] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Its been a month My work is totally blocked because of this issue and github support won't even respond to my ticket even after month.
And i have no idea what to do, creating a new account is not an option.
What the hell should i do ? 
TicketID: #2620051

**Comment by benogoulart** [comment]
Hi,

I'm sorry to hear you're experiencing this. Waiting a month without a response is understandably frustrating, especially if it's preventing you from working.

Unfortunately, if your account is restricted or marked as spam, only GitHub Support can review and resolve the issue. The GitHub Community doesn't have access to support tickets or the ability to escalate them.

Since you've already opened a ticket and have a Ticket ID, the best course of action is to continue with that existing ticket rather than creating new ones, as multiple requests generally won't speed up the review process.

Hopefully, a GitHub staff member can confirm that your ticket is in the queue or provide an update on its status.

I hope your case is resolved as soon as possible.
